### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by Iqbal-Elham**
## Contact Section
This is my fourth project in Microverse, and I have added a contact form for my personal portfolio mobile version website.
The requirements needed:
- Using the Html/CSS
- Using the linter for free of errors and beautiful coding.
- Using GitHub flow.
- Finally documented in a professional way.

### CODE REVIEW

**Review by anagudelogu — COMMENTED** [review]
Hi @Iqbal-Elham, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have any commits from your partner, please take in mind that both of you should contribute to each one's code in this milestone. 🤖 

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,229 +1,257 @@
...
+        </li>
+      </ul>
+    </div>
+  </section>
+  <footer class="contact">
+    <h1>
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </h1>
+    <form action="" method="post" class="contact-form">
```
Comment (ridwanediallo) [inline index.html]: - [x] Great job so far 👏. However, I think your `form` should have an `action`  value that is similar to  `action="https://formspree.io/f/{form_id} " `
- [x] What you need first is to create your account at [formspree](https://formspree.io/html/) and then generate the `form_id`. 
- [x] In your `action` value replace `{form_id}` with the `id` you generated from  [formspree](https://formspree.io/html/). 👌

- [x] Lastly, please make that a user can successfully submit a `form`. After that is done you can proudly submit it for review. 🤗

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @Iqbal-Elham , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey @Iqbal-Elham :wave:

Great work on implementing the requirements. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following GitFlow
:heavy_check_mark: Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**